rpcs3/Emu/System.cpp

```diff
@@ -439,6 +443,20 @@ void Emulator::Load(bool add_only)
 		}
 
 		LOG_NOTICE(LOADER, "Used configuration:\n%s\n", g_cfg.to_string());
+		
+		// Set RTM usage
+		g_use_rtm = utils::has_rtm() && (utils::is_haswell_or_broadwell() == utils::ModelNotHaswellBroadwell || g_cfg.core.enable_TSX);
+		if (utils::has_rtm())
+		{
+			if (g_use_rtm)
+			{
+				LOG_NOTICE(GENERAL, "TSX enabled");
```

Comment:
these are more or less useless, since you can see this on the log anyway

Reply:
I'll change it so the log is only displayed on Haswell and Broadwell CPUs.